src/main/java/org/fogbowcloud/arrebol/execution/docker/DockerCommandExecutor.java

```diff
@@ -0,0 +1,57 @@
+package org.fogbowcloud.arrebol.execution.docker;
+
+import org.apache.log4j.Logger;
+import org.fogbowcloud.arrebol.execution.docker.request.ExecInstanceResult;
+import org.fogbowcloud.arrebol.execution.docker.request.WorkerDockerRequestHelper;
+
+import static java.lang.Thread.sleep;
+
+public class DockerCommandExecutor {
```

Comment:
overall, I thinks it good code. But we need some doc in the class (it is a key piece of our codebase). the reader should know is is polling-based. Also, you should tell the reader something about the async method

Reply:
I added a documentation for the two methods of the class.